NotesApi/V1/Domain/Queries/GetNotesByTargetIdQuery.cs

```diff
@@ -0,0 +1,16 @@
+using Microsoft.AspNetCore.Mvc;
+using System;
+using System.ComponentModel.DataAnnotations;
+
+namespace NotesApi.V1.Domain.Queries
```

Comment:
Would this not be in the Boundary/Request folder

Reply:
It could be, but it could equally be regarded as _a query against the domain_ that could be initiated from an API controller just as well as from some other means (a message bus message for instance). 
In each case the mechanism by which the query is triggered is different, but the nature of the query against the domain remains the same.